sdk/remoterendering/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/_generated/models/_models_py3.py

```diff
@@ -503,17 +503,17 @@ def __init__(
         **kwargs
     ):
         super(RenderingSession, self).__init__(**kwargs)
-        self.id = id
-        self.arr_inspector_port = None
-        self.handshake_port = None
-        self.elapsed_time_minutes = None
-        self.hostname = None
-        self.lease_time_minutes = None
-        self.size = size
-        self.status = status
-        self.teraflops = None
-        self.error = None
-        self.created_on = None
+        self.id: str = id
+        self.arr_inspector_port: int = None
+        self.handshake_port:int = None
+        self.elapsed_time_minutes: int = None
+        self.hostname: str = None
+        self.lease_time_minutes: int = None
+        self.size: str = size
+        self.status: Union[str, RenderingSessionStatus] = status
+        self.teraflops: float = None
+        self.error: RemoteRenderingError = None
+        self.created_on: datetime.datetime = None
```

Comment:
I tried regenerating the code, but the results were vastly different from what I see right now (there wasn't even a _models_py3.py file anymore). I don't think it is good, that it differs this much, but I think regenerating everything should be its own thing.

I moved the main swagger file into this repository though, as it was still on a personal git account, which is of course very bad. Is this fine or should I move it to https://github.com/Azure/azure-rest-api-specs? (I just saw, that many other projects use that repository for the swagger files)

Reply:
Your swagger spec should be committed to the azure-rest-api-specs repo, that is the source of truth for your service. Since it seems we might have to regenerate in a different PR, let's revert any changes made to the generated code in this one.

Reply:
I removed the changes to the generated things, but had to add the casts again otherwise the pipelines will fail.

I also found that the swagger file was already in the specs repository, but under Mixedreality/data-plane, which is a very bad naming for the path... But hey, it makes all of this easier.